examples/powershell/vmware-cloud-ngw-ms-teams/ngw-teams-config.json

```diff
@@ -0,0 +1,3 @@
+{
+    TEAMS_WEBHOOK_URL: "FILL-ME-IN"
```

Comment:
```suggestion
    "TEAMS_WEBHOOK_URL": "FILL-ME-IN"
```

Reply:
Adding quotes so that a user will paste the URL with these quotes (invalid JSON otherwise)

Reply:
Good eye! Interestingly, PowerShell was able to parse this w/o issues :)

Reply:
Oh, no :)